src/server/routes/search/search.js

```diff
@@ -24,7 +24,8 @@ const idFromXrefs = ( xrefLinks, namespace ) => {
 const sanitize = ( rawQuery, maxLength = QUERY_MAX_CHARS ) => rawQuery.trim().substring( 0, maxLength );
 const splitOnWhitespace = tokens => _.flatten( tokens.map( t => t.split(/\s+/) ) );
 const splitOnCommas = queryString => queryString.split(/,/).map( t => t.trim() );
-const tokenize = ( rawQuery, maxNum = QUERY_MAX_TOKENS ) => splitOnWhitespace( splitOnCommas( rawQuery ) ).slice( 0, maxNum );
+const dropQuotes = tokens => tokens.map( t => t.replace(/^['"]|['"]$/g, '') );
```

Comment:
Would `/['"]/g` suffice?  That would make the regex easier to read.

Reply:
yeah.